### DISCUSSION TITLE
GitHub sponsor profile for my org has not been reviewed since last 10 days

### DISCUSSION THREAD

**Discussion by WasiqB** [community/community#101645] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Normally how long does it takes for the organization sponsor profile to get reviewed?

My profile https://github.com/sponsors/Learn-Test-Automation has been submitted for review on 14th Jan 2024 but till now there is no progress.

Can someone look into it?

**Comment by n31t** [comment]
![Безымянный](https://github.com/community/community/assets/112765709/9bbe4921-cb38-44d8-9bff-9fb70cd6bb53)
It looks fine for me

**Reply by WasiqB** [reply]
My profile got approved on 26th Jan 2024
